### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Delete_DeletePopup_ScreenReader: Voice Over is not announcing the status message after deleting the blob container

### ISSUE THREAD

**Issue by shreyashi227** [microsoft/AzureStorageExplorer#2625]
​**GitHub Tags:** #A11y_AzureToolsStorageExplorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp; #MacOS; #Benchmark; #A11ySev2; #WCAG4.1.3; #Azure Tools Storage Explorer; VoiceOver

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.12.0
Operating System :Mac

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Once Completed, Right click on uploaded blob and Select "Delete" and hit enter.
7. Open Voice Over and check if Voice Over is announcing the status message after deleting the blob container

**Actual:**
Voice Over is not announcing the status message after deleting the blob container

**Similar issue has been observed for below Scenarios:**
1) Upload_UploadFolder
2) BlobOperations_CloneWithNewNameBlob
3) Upload_UploadFiles
4) ChangeAccessTier_UpdateAccessTier

**Expected:**
Voice Over should announce the status message after deleting the blob container

**User Impact:**
Screen reader users will not get the status message of deleting the blob container

**Comment by shreyashi227** [comment]
We have uploaded repro video of this bug in [OneDrive](https://microsoft-my.sharepoint.com/personal/v-subha_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fv%2Dsubha%5Fmicrosoft%5Fcom%2FDocuments%2FMicrosoft%20Azure%20Tools%20Storage%20Explorer%20Attachments&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOmY6L3Avdi1zdWJoYS9FdnhIZGcxemtOOVBrbkYxdU94bk9pUUJlanB4S1lnOW4yMUR5ZGdRX29tMGV3P3J0aW1lPWJNTVh6dTJ1MTBn) link with name "2625_A11y_AzureToolsStorageExplorer_Delete_DeletePopup_ScreenReader". Please refer the video for reference.
Please let us know if there is any problem in accessing the link

**Comment by MRayermannMSFT** [comment]
Chitrarupa said this was fixed in email from 10/19.

**Comment by Ashutoshsahoo164** [comment]
GitHubTag: #BM-Azure Tools Storage Explorer-MAC-Aug2020

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by InduPriya1805** [comment]
GitHubTag:#BM-Azure Tools Storage Explorer-MAC-Aug2020;
